static/app/components/deprecatedforms/passwordField.tsx

```diff
@@ -27,17 +27,15 @@ export default class PasswordField extends InputField<Props, State> {
     this.state = {...this.state, editing: false};
   }
 
-  UNSAFE_componentWillReceiveProps(nextProps: Props) {
+  componentDidUpdate(prevProps: Props) {
     // close edit mode after successful save
     // TODO(dcramer): this needs to work with this.context.form
     if (
       this.props.formState &&
-      this.props.formState === FormState.SAVING &&
-      nextProps.formState === FormState.READY
+      prevProps.formState === FormState.SAVING &&
+      this.props.formState === FormState.READY
```

Comment:
Maybe add a check that `editing` isn't already `false`?

Reply:
it will just be a no-op then, I think it doesn't matter

Reply:
I think this one is good to go looking over it again

Reply:
iirc it's only primitives w/ useState hook that are a no-op, but if you've check it and it seems to work then 👍